create/templates/shared/shopify.ui.extension.toml

```diff
@@ -1,2 +1,2 @@
-type = "{{ .NormalizedType }}"
+type = "{{ .Type }}"
```

Comment:
This needs to be `.ExternalType`

Reply:
Do we need to have both external type and type or just external type is enough? As long as I know the external type is used just for displaying to the public